Dockerfile

```
@@ -0,0 +1,26 @@
+FROM python:3.9-slim
+
+WORKDIR /app
+RUN pip install --upgrade pip
+COPY requirements.txt requirements.txt
+RUN pip3 install --no-cache-dir -r requirements.txt
+RUN apt-get update && \
```

Hadolint (2.12.0)

[info] Avoid additional packages by specifying `--no-install-recommends`

(DL3015)